[2022-05-30 13:05:08 +0000] [24955] [DEBUG] Current configuration:
  config: ./gunicorn.conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/DjangoApp/mywebsite.sock']
  backlog: 2048
  workers: 3
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/DjangoApp
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 33
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: DjangoApp
  default_proc_name: DjangoApp.wsgi:application
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f7d15da8510>
  on_reload: <function OnReload.on_reload at 0x7f7d15da8620>
  when_ready: <function WhenReady.when_ready at 0x7f7d15da8730>
  pre_fork: <function Prefork.pre_fork at 0x7f7d15da8840>
  post_fork: <function Postfork.post_fork at 0x7f7d15da8950>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f7d15da8a60>
  worker_int: <function WorkerInt.worker_int at 0x7f7d15da8b70>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f7d15da8c80>
  pre_exec: <function PreExec.pre_exec at 0x7f7d15da8d90>
  pre_request: <function PreRequest.pre_request at 0x7f7d15da8ea0>
  post_request: <function PostRequest.post_request at 0x7f7d15da8f28>
  child_exit: <function ChildExit.child_exit at 0x7f7d15dc10d0>
  worker_exit: <function WorkerExit.worker_exit at 0x7f7d15dc11e0>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f7d15dc12f0>
  on_exit: <function OnExit.on_exit at 0x7f7d15dc1400>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2022-05-30 13:05:08 +0000] [24955] [INFO] Starting gunicorn 20.1.0
[2022-05-30 13:05:08 +0000] [24955] [DEBUG] Arbiter booted
[2022-05-30 13:05:08 +0000] [24955] [INFO] Listening at: unix:/home/debian/DjangoApp/mywebsite.sock (24955)
[2022-05-30 13:05:08 +0000] [24955] [INFO] Using worker: sync
[2022-05-30 13:05:08 +0000] [24958] [INFO] Booting worker with pid: 24958
[2022-05-30 13:05:08 +0000] [24959] [INFO] Booting worker with pid: 24959
[2022-05-30 13:05:08 +0000] [24960] [INFO] Booting worker with pid: 24960
[2022-05-30 13:05:08 +0000] [24955] [DEBUG] 3 workers
[2022-05-30 13:05:18 +0000] [24955] [INFO] Handling signal: term
[2022-05-30 13:05:18 +0000] [24958] [INFO] Worker exiting (pid: 24958)
[2022-05-30 13:05:18 +0000] [24959] [INFO] Worker exiting (pid: 24959)
[2022-05-30 13:05:18 +0000] [24960] [INFO] Worker exiting (pid: 24960)
[2022-05-30 13:05:19 +0000] [24955] [INFO] Shutting down: Master
[2022-05-30 13:05:19 +0000] [24964] [DEBUG] Current configuration:
  config: ./gunicorn.conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/DjangoApp/mywebsite.sock']
  backlog: 2048
  workers: 3
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/DjangoApp
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 33
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: DjangoApp
  default_proc_name: DjangoApp.wsgi:application
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fc38c263510>
  on_reload: <function OnReload.on_reload at 0x7fc38c263620>
  when_ready: <function WhenReady.when_ready at 0x7fc38c263730>
  pre_fork: <function Prefork.pre_fork at 0x7fc38c263840>
  post_fork: <function Postfork.post_fork at 0x7fc38c263950>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fc38c263a60>
  worker_int: <function WorkerInt.worker_int at 0x7fc38c263b70>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fc38c263c80>
  pre_exec: <function PreExec.pre_exec at 0x7fc38c263d90>
  pre_request: <function PreRequest.pre_request at 0x7fc38c263ea0>
  post_request: <function PostRequest.post_request at 0x7fc38c263f28>
  child_exit: <function ChildExit.child_exit at 0x7fc38c27b0d0>
  worker_exit: <function WorkerExit.worker_exit at 0x7fc38c27b1e0>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fc38c27b2f0>
  on_exit: <function OnExit.on_exit at 0x7fc38c27b400>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2022-05-30 13:05:19 +0000] [24964] [INFO] Starting gunicorn 20.1.0
[2022-05-30 13:05:19 +0000] [24964] [DEBUG] Arbiter booted
[2022-05-30 13:05:19 +0000] [24964] [INFO] Listening at: unix:/home/debian/DjangoApp/mywebsite.sock (24964)
[2022-05-30 13:05:19 +0000] [24964] [INFO] Using worker: sync
[2022-05-30 13:05:19 +0000] [24967] [INFO] Booting worker with pid: 24967
[2022-05-30 13:05:19 +0000] [24968] [INFO] Booting worker with pid: 24968
[2022-05-30 13:05:19 +0000] [24969] [INFO] Booting worker with pid: 24969
[2022-05-30 13:05:19 +0000] [24964] [DEBUG] 3 workers
[2022-05-30 16:56:02 +0000] [24964] [INFO] Handling signal: term
Traceback (most recent call last):
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 224, in run
    handler()
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 257, in handle_term
    raise StopIteration
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/debian/DjangoApp/venv/bin/gunicorn", line 10, in <module>
    sys.exit(run())
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/app/wsgiapp.py", line 67, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 231, in run
    super().run()
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/app/base.py", line 72, in run
    Arbiter(self).run()
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 227, in run
    self.halt()
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 342, in halt
    self.stop()
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/arbiter.py", line 382, in stop
    sock.close_sockets(self.LISTENERS, unlink)
  File "/home/debian/DjangoApp/venv/lib/python3.7/site-packages/gunicorn/sock.py", line 212, in close_sockets
    os.unlink(sock_name)
FileNotFoundError: [Errno 2] No such file or directory: '/home/debian/DjangoApp/mywebsite.sock'
[2022-05-30 16:57:06 +0000] [488] [DEBUG] Current configuration:
  config: ./gunicorn.conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/DjangoApp/mywebsite.sock']
  backlog: 2048
  workers: 3
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/DjangoApp
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 33
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: DjangoApp
  default_proc_name: DjangoApp.wsgi:application
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7f4aa65ee510>
  on_reload: <function OnReload.on_reload at 0x7f4aa65ee620>
  when_ready: <function WhenReady.when_ready at 0x7f4aa65ee730>
  pre_fork: <function Prefork.pre_fork at 0x7f4aa65ee840>
  post_fork: <function Postfork.post_fork at 0x7f4aa65ee950>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7f4aa65eea60>
  worker_int: <function WorkerInt.worker_int at 0x7f4aa65eeb70>
  worker_abort: <function WorkerAbort.worker_abort at 0x7f4aa65eec80>
  pre_exec: <function PreExec.pre_exec at 0x7f4aa65eed90>
  pre_request: <function PreRequest.pre_request at 0x7f4aa65eeea0>
  post_request: <function PostRequest.post_request at 0x7f4aa65eef28>
  child_exit: <function ChildExit.child_exit at 0x7f4aa66070d0>
  worker_exit: <function WorkerExit.worker_exit at 0x7f4aa66071e0>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7f4aa66072f0>
  on_exit: <function OnExit.on_exit at 0x7f4aa6607400>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2022-05-30 16:57:06 +0000] [488] [INFO] Starting gunicorn 20.1.0
[2022-05-30 16:57:06 +0000] [488] [DEBUG] Arbiter booted
[2022-05-30 16:57:06 +0000] [488] [INFO] Listening at: unix:/home/debian/DjangoApp/mywebsite.sock (488)
[2022-05-30 16:57:06 +0000] [488] [INFO] Using worker: sync
[2022-05-30 16:57:06 +0000] [508] [INFO] Booting worker with pid: 508
[2022-05-30 16:57:06 +0000] [512] [INFO] Booting worker with pid: 512
[2022-05-30 16:57:06 +0000] [529] [INFO] Booting worker with pid: 529
[2022-05-30 16:57:06 +0000] [488] [DEBUG] 3 workers
[2022-07-11 15:26:53 +0000] [491] [DEBUG] Current configuration:
  config: ./gunicorn.conf.py
  wsgi_app: None
  bind: ['unix:/home/debian/DjangoApp/mywebsite.sock']
  backlog: 2048
  workers: 3
  worker_class: sync
  threads: 1
  worker_connections: 1000
  max_requests: 0
  max_requests_jitter: 0
  timeout: 30
  graceful_timeout: 30
  keepalive: 2
  limit_request_line: 4094
  limit_request_fields: 100
  limit_request_field_size: 8190
  reload: False
  reload_engine: auto
  reload_extra_files: []
  spew: False
  check_config: False
  print_config: False
  preload_app: False
  sendfile: None
  reuse_port: False
  chdir: /home/debian/DjangoApp
  daemon: False
  raw_env: []
  pidfile: None
  worker_tmp_dir: None
  user: 0
  group: 33
  umask: 0
  initgroups: False
  tmp_upload_dir: None
  secure_scheme_headers: {'X-FORWARDED-PROTOCOL': 'ssl', 'X-FORWARDED-PROTO': 'https', 'X-FORWARDED-SSL': 'on'}
  forwarded_allow_ips: ['127.0.0.1']
  accesslog: None
  disable_redirect_access_to_syslog: False
  access_log_format: %(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"
  errorlog: -
  loglevel: debug
  capture_output: False
  logger_class: gunicorn.glogging.Logger
  logconfig: None
  logconfig_dict: {}
  syslog_addr: udp://localhost:514
  syslog: False
  syslog_prefix: None
  syslog_facility: user
  enable_stdio_inheritance: False
  statsd_host: None
  dogstatsd_tags: 
  statsd_prefix: 
  proc_name: DjangoApp
  default_proc_name: DjangoApp.wsgi:application
  pythonpath: None
  paste: None
  on_starting: <function OnStarting.on_starting at 0x7fd537f048c8>
  on_reload: <function OnReload.on_reload at 0x7fd537f049d8>
  when_ready: <function WhenReady.when_ready at 0x7fd537f04ae8>
  pre_fork: <function Prefork.pre_fork at 0x7fd537f04bf8>
  post_fork: <function Postfork.post_fork at 0x7fd537f04d08>
  post_worker_init: <function PostWorkerInit.post_worker_init at 0x7fd537f04e18>
  worker_int: <function WorkerInt.worker_int at 0x7fd537f04f28>
  worker_abort: <function WorkerAbort.worker_abort at 0x7fd537f1c0d0>
  pre_exec: <function PreExec.pre_exec at 0x7fd537f1c1e0>
  pre_request: <function PreRequest.pre_request at 0x7fd537f1c2f0>
  post_request: <function PostRequest.post_request at 0x7fd537f1c378>
  child_exit: <function ChildExit.child_exit at 0x7fd537f1c488>
  worker_exit: <function WorkerExit.worker_exit at 0x7fd537f1c598>
  nworkers_changed: <function NumWorkersChanged.nworkers_changed at 0x7fd537f1c6a8>
  on_exit: <function OnExit.on_exit at 0x7fd537f1c7b8>
  proxy_protocol: False
  proxy_allow_ips: ['127.0.0.1']
  keyfile: None
  certfile: None
  ssl_version: 2
  cert_reqs: 0
  ca_certs: None
  suppress_ragged_eofs: True
  do_handshake_on_connect: False
  ciphers: None
  raw_paste_global_conf: []
  strip_header_spaces: False
[2022-07-11 15:26:53 +0000] [491] [INFO] Starting gunicorn 20.1.0
[2022-07-11 15:26:53 +0000] [491] [DEBUG] Arbiter booted
[2022-07-11 15:26:53 +0000] [491] [INFO] Listening at: unix:/home/debian/DjangoApp/mywebsite.sock (491)
[2022-07-11 15:26:53 +0000] [491] [INFO] Using worker: sync
[2022-07-11 15:26:53 +0000] [529] [INFO] Booting worker with pid: 529
[2022-07-11 15:26:53 +0000] [531] [INFO] Booting worker with pid: 531
[2022-07-11 15:26:53 +0000] [537] [INFO] Booting worker with pid: 537
[2022-07-11 15:26:53 +0000] [491] [DEBUG] 3 workers
